applejack/conf/products/central-configuration.yml

```diff
@@ -78,6 +78,8 @@ context:
       value: ""
     - key: XLD_TASK_QUEUE_PASSWORD
       value: ""
+    - key: XLD_TASK_QUEUE_IS_QUORUM
+      value: "false"          
```

Comment:
This env var is only for rabbitmq, so why not to have rabbitmq in the name. And intead of using bool value use for example name of the queue type, for example:

   - key: XLD_TASK_RABBITMQ_QUEUE_TYPE
      value: "normal" or "quorum"